File: Linked List/Python/GFG - PRINT LINKED LIST.py
# GFG - Print Linked List
# QUESTION LINK - https://www.geeksforgeeks.org/problems/print-linked-list-elements/1

class Solution:
    def printList(self, head):
        temp = head
        while temp is not None:
            print(temp.data, end=" ")
            temp = temp.next

File: Linked List/Java/GFG - PRINT LINKED LIST.java
// GFG - Print Linked List
// QUESTION LINK - https://www.geeksforgeeks.org/problems/print-linked-list-elements/1

class Solution {
    void printList(Node head) {
        Node temp = head;
        while (temp != null) {
            System.out.print(temp.data + " ");
            temp = temp.next;
        }
    }
}
